src/lib/components/search.svelte

```diff
@@ -0,0 +1,3 @@
+ <label>
+    <input class="input text-black p-6 placeholder-black" type="text" placeholder="Dies ist der Surchtext..." />
```

Comment:
I think it's the third time that I'm suggesting this:
```suggestion
    <input class="input text-black p-6 placeholder-black" type="text" placeholder="Dies ist der Suchtext..." />
```

Reply:
also I please don't use black as colors but primary-500 (as I suggested 2 weeks ago):
```suggestion
    <input class="input text-primary-500 p-6 placeholder-primary-500" type="text" placeholder="Dies ist der Suchtext..." />
```

Reply:
Sorry about that...